C#Backend/InpatientTherapySchedulingProgram/Controllers/HoursWorkedController.cs

```diff
@@ -6,39 +6,49 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using InpatientTherapySchedulingProgram.Models;
+using InpatientTherapySchedulingProgram.Services.Interfaces;
+using InpatientTherapySchedulingProgram.Exceptions.HoursWorkedExceptions;
+using InpatientTherapySchedulingProgram.Exceptions.UserExceptions;
 
 namespace InpatientTherapySchedulingProgram.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class HoursWorkedController : ControllerBase
     {
-        private readonly CoreDbContext _context;
+        private readonly IHoursWorkedService _service;
 
-        public HoursWorkedController(CoreDbContext context)
+        public HoursWorkedController(IHoursWorkedService service)
         {
-            _context = context;
+            _service = service;
         }
 
-        // GET: api/HoursWorkeds
-        [HttpGet]
-        public async Task<ActionResult<IEnumerable<HoursWorked>>> GetHoursWorked()
+        // GET: api/HoursWorkeds/5
+        [HttpGet("{id}")]
```

Comment:
I believe the "id" name has to match the parameter in the function call. I would test this in Postman but pretty sure you just have to rename id to hoursWorkedId.